README.md

```diff
@@ -1,25 +1,38 @@
 # dp-search-api
 
-Digital Publishing Search API
+Dissemination Service Search API
 
 A Go application microservice to provide query functionality on the ONS Website
 
 ## Getting started
 
 Set up dependencies locally as follows:
 
-* In [dp-compose](https://github.com/ONSdigital/dp-compose) run `docker-compose up -d` to run ElasticSearch 7.10
-    * dp-compose will run Elasticsearch 7.10 on port 11200 to not conflict with ES 2.2 running on port 9200
-* If using the POST /search endpoint then authorisation for this requires running Vault and Zebedee as follows:
-* In any directory run `vault server -dev` as Zebedee has a dependency on Vault
-* In the zebedee directory run `./run.sh` to run Zebedee
+* See our [Docker Compose Search Stack](https://github.com/ONSdigital/dp-compose/tree/main/v2/stacks/search) to run all services end to end.
 
-Then run `make debug`
+Alternatively you can run individual services:
```

Comment:
There are many individual services that can be run, not mentioned, so I would say something like this:
```suggestion
Alternatively you can run individual services, which are listed in the frontend.yml and backend.yml files in the stack.
```